tests/unit/views/production/ProductionCastCredits.spec.js

```diff
@@ -194,6 +205,9 @@ describe('CastCreditsContainer', function () {
       propsData: {
         production: production,
       },
+      stubs: {
```

Comment:
Instead of doing this manually, use the `mountOptionsWithRouter` to wrap the options or `mountWithRouterMock` like https://github.com/BristolSTA/uobtheatre-web/blob/main/tests/unit/views/venues/venue.spec.js#L36